torch/csrc/Generator.cpp

```diff
@@ -69,7 +69,7 @@ static PyObject * THPGenerator_getState(THPGenerator *self)
   HANDLE_TH_ERRORS
   THGenerator *generator = THPGenerator_TH_CData(self);
   Variable var = torch::empty({0}, at::device(at::kCPU).dtype(at::kByte));
-  THByteTensor_getRNGState(generator, (THByteTensor*)(var.data().unsafeGetTensorImpl()));
+  THByteTensor_getRNGState(generator, (THByteTensor*)(var.unsafeGetTensorImpl()));
```

Comment:
After this patch, Variable `var`'s tensor data is stored in `var.unsafeGetTensorImpl()` instead of `var.data_.unsafeGetTensorImpl()`, and `var` doesn't require grad, so it's okay to use `var` instead of `var.data()` here.